recipes/folly/all/test_v1_package/CMakeLists.txt

```diff
@@ -4,14 +4,5 @@ project(test_package CXX)
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup(TARGETS)
 
-find_package(folly CONFIG REQUIRED)
-
-add_executable(${PROJECT_NAME} test_package.cpp)
-target_link_libraries(${PROJECT_NAME} Folly::folly)
-
-
-if (${FOLLY_VERSION} VERSION_LESS "2021.07.20.00")
-    set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 14)
-else()
-    set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 17)
-endif()
+add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../test_package
```

Comment:
Feel free to remove the `test_v1_package` folder from the PR - it's not required anymore.

Reply:
(sorry I've been disconnected for a while) are test_v1_packages being removed already? Is there any problem if I keep it for this recipe?

I personally cannot migrate to Conan 2 yet and having a test_v1_package would be handy for me.

Reply:
@Hopobcn You still can use `CMakeToolchain` and `CMakeDeps` generators in Conan 1.x. They are much better than older `cmake` and `cmake_find_package`. Docs: https://docs.conan.io/1/reference/conanfile/tools/cmake/cmaketoolchain.html

Reply:
@uilianries yes we use Conan 1.x with CMakeToolchain/CMakeDeps already but in the past I had problems trying to use `test_package` with conan 1.x CMakeToolchain/CMakeDeps so we use test_v1_package mainly. Our blockers are setting up new infrastructure for Conan 2 (artifactory remotes, docker images) [easy], CLI changes to be applied at couple of tens of projects (cryface)... and that we have to catch-up with latest CCI (cryface x100).